Tests/test_hook_manager.py

```diff
@@ -0,0 +1,90 @@
+import unittest
+import os
+import mock as mock
+import inspect
+import shutil
+
+from kalliope.core.Models import Singleton
+
+from kalliope.core.ConfigurationManager import SettingLoader
+
+from kalliope.core import HookManager
+from kalliope.core.Models.Settings import Settings
+
+
+class TestInit(unittest.TestCase):
```

Comment:
What happens when a synapse does not exists in the list ? 
Can we add a test to catch exception if the synapse does not exists in the brain ?

Reply:
good idea !

Reply:
and it should work properly as [Hook class is based on the synapse launcher](https://github.com/kalliope-project/kalliope/blob/feature/hooks/kalliope/core/HookManager.py#L68)